xml/libvirt_networking.xml

```diff
@@ -8,958 +8,978 @@
+  There are two common network configurations to provide a &vmguest; with a
+  network connection:
  </para>
  <itemizedlist>
   <listitem>
    <para>
-    A <emphasis>virtual network</emphasis> for the guest
+    A <emphasis>network bridge</emphasis> acting as a Layer 2 switch
    </para>
   </listitem>
   <listitem>
    <para>
-    A <emphasis>network bridge</emphasis> over a host's physical network
-    interface that the guest can use
+    A <emphasis>virtual network</emphasis> managed by &libvirt; with Layer 3
+    forwarding enabled
    </para>
   </listitem>
  </itemizedlist>
- <sect1 xml:id="libvirt-networks-virtual">
-  <title>Virtual Networks</title>
-
-  <para>
-   A virtual network is a computer network which does not consist of a physical
-   network link, but rather uses a virtual network link. Each host can have
-   several virtual networks defined. Virtual networks are based on virtual
-   devices that connect virtual machines inside a hypervisor. They allow
-   outgoing traffic translated to the LAN and are provided with DHCP and DNS
-   services. Virtual networks can be either <emphasis>isolated</emphasis>, or
-   <emphasis>forwarded</emphasis> to a physical network.
-  </para>
-
-  <para>
-   Guests inside an <emphasis>isolated</emphasis> virtual network can
-   communicate with each other, but cannot communicate with guests outside the
-   virtual network. Also, guests not belonging to the isolated virtual network
-   cannot communicate with guests inside.
-  </para>
+ <sect1 xml:id="libvirt-networks-bridged">
+  <title>Network Bridge</title>
 
   <para>
-   On the other hand, guests inside a <emphasis>forwarded</emphasis> (NAT,
-   network address translation) virtual network can make any outgoing network
-   connection they request. Incoming connections are allowed from &vmhost;, and
-   from other guests connected to the same virtual network. All other incoming
-   connections are blocked by <systemitem>iptables</systemitem> rules.
+   The network bridge configuration provides a Layer 2 switch for &vmguest;s,
+   switching Layer 2 Ethernet packets between ports on the bridge based on MAC
+   addresses associated with the ports. This gives the &vmguest; Layer 2 access
+   to the &vmhost;'s network. This configuration is analogous to connecting the
+   &vmguest;'s virtual Ethernet cable into a hub that is shared with the host
+   and other &vmguest;s running on the host. The configuration is often referred
+   to as <emphasis>shared physical device</emphasis>.
   </para>
 
   <para>
-   A standard libvirt installation on &productname; already comes with a
-   predefined virtual network providing DHCP server and network address
-   translation (NAT) named "default".
+   The network bridge configuration is the default configuration of
+   &productname; when configured as a &kvm; or &xen; hypervisor. It is the
+   preferred configuration when you simply want to connect &vmguest;s to the
+   &vhhost;'s LAN.
   </para>
```

Comment:
I like this change, from a user perspective it is much clearer what the goal of the configuration is.

Reply:
Thanks. I think my latest tweak provides another small, incremental improvement.